docs/logical_data_model.md

```diff
@@ -96,6 +96,16 @@ class TopicGoal {
   * updatedAt : timestamp
 }
 
+class NextSteps {
+  * id: integer
+  * activityReportId: integer(32) REFERENCES public.ActivityReport.id
+  * userId: integer(32) REFERENCES public.Users.id
```

Comment:
Is there a plan for using the `userId` for anything? It feels unnecessary at this point so I'd prefer to remove it unless there's a known need.